package.json

```diff
@@ -8,9 +8,11 @@
     "start": "next start",
     "lint": "next lint",
     "format": "prettier --write \"**/*.{md,html,mjml,json,yml,css,scss,ts,tsx,js,jsx}\"",
-    "lint:fix": "eslint '**/*.{ts,tsx}' --fix"
+    "lint:fix": "eslint '**/*.{ts,tsx}' --fix",
+    "tsc": "tsc"
   },
   "dependencies": {
+    "@apollo/client": "^3.11.1",
```

Comment:
I was using `urql` to do graphql requests here, but definitely not married to it. Do you have any preferences towards using apollo?

Reply:
I don't have a strong preference towards using Apollo over urql. Apollo is more widely used and has a rich ecosystem, but urql is lightweight and has some nice features like built-in hooks for easier integration with React. I'm comfortable with either, so I'm happy to go with any. If we decide to stick with urql, I can adapt the implementation accordingly.

Reply:
Changed to urql @ribeirojose